src/components/ReaderLayout/ReaderLayout.tsx

```diff
@@ -3,21 +3,26 @@ import * as DS from "@nypl/design-system-react-components";
 import { breadcrumbTitles } from "~/src/constants/labels";
 import { ApiLink, LinkResult } from "~/src/types/LinkQuery";
 import IFrameReader from "../IFrameReader/IFrameReader";
-import WebpubViewer from "../WebpubViewer/WebpubViewer";
 import EditionCardUtils from "~/src/util/EditionCardUtils";
 import Layout from "../Layout/Layout";
 import * as gtag from "../../lib/Analytics";
-import { truncateStringOnWhitespace } from "~/src/util/Util";
+import { formatUrl, truncateStringOnWhitespace } from "~/src/util/Util";
 import { MAX_TITLE_LENGTH } from "~/src/constants/editioncard";
-
+import dynamic from "next/dynamic";
+import { useRouter } from "next/router";
+import { MediaTypes } from "~/src/constants/mediaTypes";
+import { proxyUrl } from "~/src/lib/api/SearchApi";
+const WebReader = dynamic(() => import("@nypl/web-reader"), { ssr: false });
 //The NYPL wrapper that wraps the Reader pages.
 const ReaderLayout: React.FC<{ linkResult: LinkResult }> = (props) => {
+  const router = useRouter();
+  const origin = router.basePath;
   const link: ApiLink = props.linkResult.data;
-
+  const url = formatUrl(link.url);
   const edition = link.work.editions[0];
 
-  const isRead = link.flags.reader;
-  const isEmbed = link.flags.embed;
+  const isEmbed = MediaTypes.embed.includes(link.media_type);
+  const isRead = MediaTypes.read.includes(link.media_type);
```

Comment:
Should we also update the `getReadOnlineLink` function so that it matches this? 

(http://34.204.45.18:3000/work/24211e77-3713-4ca9-9c0c-0c0c6cc54cc8 this work doesn't render because of the inconsistent media type checking)

Reply:
Oh this is a bug!  This should match getReadOnlineLink, I think!